File: lista 2 casa/questao 5.py
salario = float(input("Digite seu salario bruto:"))
horas = float(input("Digite suas horas tabalhadas:"))
if horas>160:
    valor_h=salario / 160
    horas2=horas - 160
    horas_t = horas2 * valor_h + (horas2 / 2)
    salario= salario + horas_t
if salario <= 800:
    print(f"Seu salario liquido é igual a {salario}")
elif salario > 800 and salario < 1600:
    salario=salario-(salario*0.13)
    print(f"seu salario liquido é de {salario}")
elif salario > 1600:
    salario=salario-(salario*0.22)
    print(f"seu salario liquido é de {salario}")
    

File: listas de exercicio/lista 2/lista 2 casa C/questao 5.c
#include <stdio.h>

int main(){
    float salario_b, v_horas, salario_l;
    int horas;
    printf("Digite seu salario bruto:");
    scanf("%f", &salario_b);
    printf("Digite suas horas trabalhadas:");
    scanf("%d", &horas);
    if (horas > 160)
    {
        v_horas = salario_b / 160;
        horas = horas - 160;
        horas = horas * v_horas + (horas / 2);
        salario_l = salario_b + horas;
    }
    
    if (salario_b <= 800)
    {
        printf("Seu salario liquido é de %.2f", salario_l);
    }
    if (800 < salario_b && salario_b < 1600)
    {
        salario_b = salario_b - (salario_b * 0.13);
        printf("Seu salario liquido é de %.2f", salario_l);
    }
    if (salario_b > 1600)
    {
        salario_b = salario_b - (salario_b * 0.22);
        printf("Seu salario liquido é de %.2f", salario_l);
    }
    
}